api/proto/letraz/v1/letraz-utils.proto

```
@@ -0,0 +1,147 @@
+syntax = "proto3";
+
+package letraz.v1;
```

Buf (1.54.0)

Files with package "letraz.v1" must be within a directory "letraz/v1" relative to root but were in directory "api/proto/letraz/v1".

(PACKAGE_DIRECTORY_MATCH)